packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -826,7 +826,7 @@ class EditableText extends StatefulWidget {
     this.readOnly = false,
     this.obscuringCharacter = '•',
     this.obscureText = false,
-    this.autocorrect = true,
+    bool? autocorrect,
```

Comment:
Do we need to update this line as well?

https://github.com/flutter/flutter/blob/c29c67ae4ce4ac5ac217e59dc09c20b967195b1f/packages/flutter/lib/src/widgets/editable_text.dart#L2296

Reply:
Thanks for pointing that out! I think `null` is the right choice, as well as `keyboardType`.

https://github.com/flutter/flutter/blob/c29c67ae4ce4ac5ac217e59dc09c20b967195b1f/packages/flutter/lib/src/widgets/editable_text.dart#L2324

Reply:
[90e9a5c](https://github.com/flutter/flutter/pull/165637/commits/90e9a5c059c6ba0e266e5763a058ca1f19b1a754)